TSAL information and circuit image

ljordan51 opened the pull request.

ljordan51 committed:
lucky jordan dreams

ljordan51 committed:
Dreams edit for word economy

ljordan51 committed:
Merge pull request #1 from olin-electric-motorsports/master

updating fork

ljordan51 committed:
TSAL ESF information and circuit image

aehoppe commented on esf_main.tex:
```diff
@@ -298,31 +298,39 @@ \subsubsection{Position in car}
 \subsubsection{Wiring/cables/connectors}
 %Describe wiring, show schematics, describe connectors and cables used and show useful data regarding the wiring.  Include gauge, voltage and temperature rating of wiring used and any fuses or other overcurrent protection used.
-TODO
+The circuitry designed has a TS-controlled optocoupler. When the TS is over 60V, zener diodes with a breakdown voltage of 58V and 3.6V respectably power the photodiode side which switches the mosfet side allowing current to flow from the GLV to power the TSAL. The frequency of the TSAL will be controlled by built in settings to flash at 2Hz.
```
*respectively

aehoppe replied:
also MOSFET should probably be capitalized, along with Zener.

aehoppe replied:
Also reference the figure using \Fref{}

ljordan51 replied:
do i need to add a label to the figure and/or put anything in the curly braces? or just type \Fref{} wherever i want to reference the figure? and what does Fref actually insert? Text with the the figure number?

aehoppe commented on esf_main.tex:
```diff
@@ -298,31 +298,39 @@ \subsubsection{Position in car}
+\caption{Schematic for the TSAL}
+\end{figure}
+
+All connections made by wires will be 20 AWG rated for 600V, 125°C and 7A, while all PCB traces will be a minimum of 12 mil, but nominally 20 mil in width. The TS voltage will be fused to 1A on the power line.
```
Good!

aehoppe commented on esf_main.tex:
```diff
@@ -298,31 +298,39 @@ \subsubsection{Position in car}
 \subsubsection{Position in car}
 %Provide CAD-renderings showing the relevant parts. Mark the parts in the rendering, if necessary.
-TODO
+The TSAL will be mounted to the underside of the highest point of the main roll hoop, per EV 4.12.3 & 4 using a robust 3D printed (Kevlar reinforced Nylon) bracket integrating the light and necessary wiring. This enclosure has not been designed yet. The PCB will be located in the master switch panel enclosure.
```
We'll want a CAD rendering for this probably.

ljordan51 replied:
do i need to request one from someone in mechanical? if so, who? if not, where can i find the rendering?

aehoppe reviewed (changes requested):
Looks good, needs a few changes though. See comments. Also, thanks for picking this up on short notice!

ljordan51 committed:
Grammar edits, figure reference, small changes

ljordan51 committed:
small fix

ljordan51 committed:
linking to appendix and datasheet

also fixed figure reference and bug with oz/ft^2 notation causing
italicized words, also added TSAL subsection in appendix

aehoppe committed:
Merge branch 'master' into master
